active/0000-anonymous-amd.md

```diff
@@ -0,0 +1,45 @@
+- Start Date: (fill me in with today's date, 2016-06-14)
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes extending the `app.import` API to consume anonymous AMD modules. It accompanies [ember-cli PR 5976](https://github.com/ember-cli/ember-cli/pull/5976).
+
+# Motivation
+
+AMD modules come in two flavors: named and anonymous. _Anonymous AMD_ is the more portable distribution format, but it typically requires preprocessing into _named AMD_ before it can be included into an application. Today, ember-cli users can add arbitrary third-party _named AMD_ modules into their application via:
+
+    app.import('/path/to/module.js');
+
+But this does not support _anonymous AMD_ modules, which is annoying because _anonymous AMD_ is the better format for library distribution and is widely used.
+
+# Detailed design
+
+In order to de-anonymize AMD, it's necessary to choose a name for the module for use within a given application. So I propose extending the:
+
+    app.import('/path/to/module.js');
+
+API with an additional argument:
+
+    app.import('/path/to/module.js', { amdModule: 'some-dep' });
```

Comment:
its worth noting, we _may_ (although quite likely we wont bother) want to expand this to support bundles of AMD modules, that may have more then 1 export that needs re-mapping. Is there a complementary API that we could use to enable that?